packages/console/src/pages/ApplicationDetails/components/AdvancedSettings.tsx

```diff
@@ -66,18 +66,9 @@ function AdvancedSettings({ applicationType, oidcConfig }: Props) {
       </FormField>
       {applicationType === ApplicationType.MachineToMachine && (
         <FormField title="application_details.enable_admin_access">
-          <Controller
-            name="isAdmin"
-            control={control}
-            render={({ field: { onChange, value } }) => (
-              <Switch
-                label={t('application_details.enable_admin_access_label')}
-                checked={value}
-                onChange={({ currentTarget: { checked } }) => {
-                  onChange(checked);
-                }}
-              />
-            )}
+          <Switch
```

Comment:
why did it worked before?

Reply:
It worked, but previously there was a React warning in the debugger console, saying that we are trying to turn an uncontrolled component into a controlled one. So I tried to fix that in #3719, but that fix causes this bug

Reply:
Discussed offline and it's good to go